src/components/FadableContent/styles.module.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../scss/variables';
+
+.shell {
+  position: relative;
+
+  &::after {
+    background-image: linear-gradient(rgba(255, 255, 255, 0), $white);
+    bottom: 0;
+    content: '';
+    height: 400px;
```

Comment:
Why is this enforced in this component? Could we configure its value somehow?

Reply:
I don't think anything would need to configure it but if that day comes we could easily add it to the component props.

Reply:
Why is it set to `400px` only then? Looks ridiculously small on my screen for example. Usually, we tend to consider the height of the window, no?

Reply:
Hmm. Could you send a screenshot? `400px` looked fine when viewing actual truncated code in `DiffView` or `CodeView`. I could switch it to `vh`, I suppose.

Reply:
I switched it to `50vh`